src/unicode/lut.zig

```diff
@@ -142,6 +142,32 @@ pub fn Tables(comptime Elem: type) type {
             return self.stage3[self.stage2[self.stage1[high] + low]];
         }
 
+        pub inline fn getInline(self: *const Self, cp: u21) Elem {
```

Comment:
future note: There's probably little downside to marking `get` as inline and removing this. We don't need to do this in this PR because I don't want to cause more churn waiting on benchmarks. But this should be as simple as testing terminal throughput (we already have a benchmark).

Reply:
I didn't include the results when I tested, but when I initially was doing some testing `inline` seemed to slow things down. I didn't test again after I fixed the problem with the lookups being optimized out.

Reply:
Re-tested and `inline` gets a percent or so speedup.